client/state/selectors/is-site-using-core-site-editor.js

```diff
@@ -12,7 +12,7 @@ import getRawSite from 'calypso/state/selectors/get-raw-site';
  * Checks if a site is using the core Site Editor
  *
  * @param {object} state  Global state tree
- * @param {number|null} siteId Site ID
+ * @param {object} siteId Site ID
```

Comment:
was this intentional?

Reply:
This PR is just a revert of the previous change that added it. Maybe @mattwiebe can provide some more context around this change.

Reply:
Gotcha, it seems like we would still want to keep this part of the change?  I would assume siteId is `number|null` and not an `object`.  Other than that everything looks good.

Reply:
Changed back in bd644082c3de669469f7586134c876657b6ab0f1.

Reply:
Missed this, but some kind of linter was complaining at me. I don't think it blocked anything.